UnityProject/Assets/Scripts/Sound/SunVox.cs

```diff
@@ -47,6 +47,8 @@ public struct sunvox_note {
 
 #if UNITY_EDITOR_OSX || UNITY_STANDALONE_OSX
   private const string LIBRARY_NAME = "sunvox";
+#elif UNITY_EDITOR_LINUX || UNITY_STANDALONE_LINUX
+  private const string LIBRARY_NAME = "sunvox";
 #elif UNITY_EDITOR_WINDOWS || UNITY_STANDALONE_WINDOWS || UNITY_STANDALONE_WIN
   private const string LIBRARY_NAME = "sunvox";
 #elif UNITY_STANDALONE_LINUX
```

Comment:
This bit won't be needed anymore (line 54) since it's covered by the elif you added